configure

```diff
@@ -3875,12 +3863,11 @@ esac
 #
 case $_backend in
 	sdl)
-		if test "$_eventrec" = auto ; then
-			_eventrec=yes
-		fi
 		;;
 	*)
-		_eventrec=no
+		if test "$_eventrec" = yes ; then
+			_eventrec=no
+		fi
```

Comment:
`_eventrec` can only be set to "yes" or "no", so I'm not sure what this change accomplishes.